### PR TITLE
add microservice roadmap

### PR DESCRIPTION
**Pull request by zackzhangkai**
Signed-off-by: zackzhang <zackzhang@yunify.com>

**What type of PR is this?**

Update roadmap for KubeSphere 3.1.0

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -25,14 +25,23 @@ KubeSphere Roadmap demonstrates a list of open source product development plans
 
 - [x] Run multiple DevOps pipelines at the same time instead of just one.[#1811](https://github.com/kubesphere/kubesphere/issues/1811)
 - [x] Clone pipeline. Users now can create exactly same pipeline from an existing one.[#3053](https://github.com/kubesphere/kubesphere/issues/3053)
+
+### microservice
+- [x] Support graph traffic direction. It will show outbound/inbound traffic direction in composed application traffic graph [#3153](https://github.com/kubesphere/kubesphere/issues/3153)
+- [x] Support Kiali graph. Users can manage istio by kiali directly. [#3106](https://github.com/kubesphere/kubesphere/issues/3106)
```
[the remark this answers is not in the document]
Comment (zackzhangkai) [reply docs/roadmap.md:52]: Yes

[the remark this answers is not in the document]
Comment (zackzhangkai) [reply docs/roadmap.md]: @zryfish Please review again.

**Comment by zryfish** [comment]
conflicts

**Comment by zackzhangkai** [comment]
> conflicts
Resolved

**Pull request merged**